src/translit_modules/ipa.js

```diff
@@ -0,0 +1,281 @@
+/* eslint no-useless-escape: 0 */
+
+const unescape = require('lodash.unescape');
+
+/**
+ * Returns a transliteration of Gurmukhi script
+ *
+ * @since 1.0.0
+ * @param {string} gurmukhi The string from to generate transliteration
+ * @returns {string} Returns a string of text
+ * @example
+ *
+ * translit('Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]');
+ * // => 'aai mil gurasikh aai mil too mayray guroo kay piaaaray ||'
```

Comment:
Could you please update the expected result?

Reply:
@ManjotS